docs/properties.md

```diff
@@ -0,0 +1,127 @@
+# Properties
+
+> Synonyms: design tokens, design variables, design constants, atoms
+
+Style properties are stored in a collection of JSON or JS module files. We usually keep them in a `properties` directory, but you can put them wherever you like, they just need to be referenced in the `source` attribute on your `config.json` file.
+
+A property is a collection of attributes that describe any fundamental/atomic visual style. Each attribute is a `key:value` pair. A property name and its value are considered a design token (or design variable/constant/atom).
+
+![Terminology for different parts of a JSON property](assets/property-definitions.png)
```

Comment:
Does this file exist? I don't see it in the PR.

Reply:
Fixed

Reply:
That one was a quick mock. Let me know if you want me to prepare a better version.